components/grid/grid.css

```diff
@@ -0,0 +1,38 @@
+.grid-container {
+  display: grid;
+  word-break: break-all;
+}
+
+.grid-2 {
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid-3 {
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
```

Comment:
Code looks fine with descriptive class names.

Reply:
Thanks for pointing it out, have refactored to use `repeat()` property of grids